configuration/concepts/total_joints.csv

```diff
@@ -0,0 +1,22 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:es,Short name:es,Description:en,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:2000
+245d28b0-d1c5-4b85-a2fd-94cdd610130e,,,Hip,Hip,Cadera,Cadera,,Misc,N/A,,,,,,,,,,,
+66e3e1d1-3daf-4ac3-8d79-85d0755abf00,,,Knee,Knee,Rodilla,Rodilla,,Misc,N/A,,,,,,,,,,,
+2ed9c8c0-1534-4d2d-a0d8-0987a9607d34,,,Joint,Joint,Articulación,Articulación,,Misc,Coded,,Shoulder; Hip; Knee,,,,,,,,,
+11cd4cc3-0d09-4771-aac3-40e0ab4342b5,,,Body Side,Body Side,Lado del cuerpo,Lado del cuerpo,,Misc,Coded,,Right; Left; Both,,,,,,,,,
+b3c11843-c85a-40f6-b087-7f59c00459db,,,Weeks,Weeks,Semanas,Semanas,,Misc,N/A,,,,,,,,,,,
+71301316-2981-4265-84e2-b45a7c050e73,,,Minutes,Minutes,Minutos,Minutos,,Misc,N/A,,,,,,,,,,,
+55ad06d0-f5d7-4bfc-9a3e-8a017047da26,,,Patient's Preference,Patient's Preference,Preferencia del paciente,Preferencia del paciente,,Misc,Coded,,Right; Left,,,,,,,,,
+31420631-a70e-4658-9aeb-fdac4a4f83dc,,,Duration of Pain,Duration of Pain,Duración del dolor,Duración del dolor,,Misc,Coded,,Hour(s); Weeks; Minutes; Months,,,,,,,,,
```

Comment:
We use 'Duration' and Time units instead of this

Reply:
Done

Reply:
Done